### PR TITLE
support SwiftUI

### PR DESCRIPTION
**Pull request by oozoofrog**
#200

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ subview.addSubview(subsub2view)
 
 > you can read details [here](https://developer.apple.com/documentation/uikit/nslayoutconstraint).
 
-- The first part is to get the necessary attributes using static values ​​defined in Anchors.
+- the first part is to get the necessary attributes using static values ​​defined in Anchors.
```
Comment (gmlwhdtjd) [inline README.md:117]: Why did you change the first letter of a sentence to lowercase?
By default, it seems correct to write in uppercase.

**Review by gmlwhdtjd — COMMENTED** [review]
The implementation itself is great.
However, I have a question about editing the readme.
Please check.

File: README.md (hunk 1 above)
Comment (oozoofrog) [reply README.md:117]: so good question in finally. i was waiting this asking. 🤣
upper case is not so old grammar. its from after printing tech. i know printing old enough too. technology had newly defined grammar.
now we have new writing tools like internet, sns, blogs, etc. 
i confess here, i support anti-autocaps movement. and now we n 
you can read what is that from here -> https://medium.com/@loganwhite1371/the-anti-autocaps-movement-its-much-more-than-another-gen-z-trend-20ba8047fc7f

ps. and its very comfort for typing.
ps1. you can see almost sentences start with lower-cased letter.

Comment (gmlwhdtjd) [reply README.md:117]: @oozoofrog I will reply via private message.

Comment (oozoofrog) [reply README.md:117]: our rules for writing README

- start character of paragraph is upper-case
- other sentences start with lower-case

**Review by gmlwhdtjd — APPROVED** [review]
